### PR TITLE
Null geom tests

### PR DESCRIPTION
**Pull request by iancboswell**
Add tests for null.SFPoint and null.SFPolygon. I took out the `SFPoint.SetCoords` method because it started to involve the `geom` library more than I'd like for this level of wrapping. It seems like the user could just `Set` a new `SFPoint` easily enough.

### CODE REVIEW

File: types/null/sf_point.go [hunk 1]
```diff
@@ -102,7 +90,7 @@ func (p *SFPoint) Null() {
 // Interfaces
 
 // IsNil implements the pyrrho/encoding IsNiler interface. It will return true
-// if j is null.
```
Comment (pyrrho) [inline types/null/sf_point.go:105]: lawl whoops. Good catch. That typo might exist in types/sf_point.go as well.

File: types/sf_polygon_test.go [hunk 2]
```diff
@@ -12,8 +12,8 @@ import (
 )
 
 var (
-	// These are all OpenGIS Simple Feature representations of an XY Polygon with
-	// X == 1.2 and Y == 2.3, converted between representations with
+	// These are all OpenGIS Simple Feature representations of the same
+	// XY Polygon, converted between representations with
```
Comment (pyrrho) [inline types/sf_polygon_test.go:16]: laaaaawl whoops. Thanks for catching that.

File: types/sf_polygon_test.go [hunk 3]
```diff
@@ -263,7 +263,8 @@ func TestSFPolygonMarshsalMapValue(t *testing.T) {
 	var data map[string]interface{}
 	var err error
 
-	wrapper = Wrapper{types.NewSFPolygonXY(testPolygonExternal, testPolygonInternal)}
+	wrapper = Wrapper{
+		types.NewSFPolygonXY(testPolygonExternal, testPolygonInternal)}
```
Comment (pyrrho) [inline types/sf_polygon_test.go]: I'm not gonna tell you to revert this change, but I don't get much value from this kind of reflowing. The fact that we're indenting with tabs makes the 80 column delimiter highly arbitrary, so... 'eh.

**Review by pyrrho — APPROVED** [review]
Looks good to me.

I only skimmed the polygon tests, because they're hella ugly. If something's been missed, that'll be on me, I think.

/gif ship it.

File: types/sf_polygon_test.go (hunk 3 above)
Comment (iancboswell) [reply types/sf_polygon_test.go]: Heh, fair enough. I don't mind reverting it.
